### PR TITLE
Bigtable: Add transport as wrapper for channel

### PR DESCRIPTION
**Pull request by sangramql**
Issue #6278

### CODE REVIEW

**Comment by sangramql** [comment]
@tseaver @crwilcox 
Please review the code

**Comment by sangramql** [comment]
@tseaver @crwilcox 
Please review the code

File: bigtable/google/cloud/bigtable/client.py [hunk 1]
```diff
@@ -61,11 +66,29 @@
...
+    def inner(self, transport_class=None):
         if self._emulator_host is None:
-            return client_class(
-                credentials=self._credentials, client_info=self._client_info
-            )
+            if transport_class is not None:
+                if self._channel is not None:
+                    # Use transport with channel
+                    transport_obj = _transport(
+                        channel=self._channel, default_class=transport_class
```
[the remark this answers is not in the document]
Comment (sangramql) [reply bigtable/google/cloud/bigtable/client.py]: Ok. I'll modify.

[the remark this answers is not in the document]
Comment (sangramql) [reply bigtable/google/cloud/bigtable/client.py]: @sduskis could you please review the initial checkin, [link](https://github.com/googleapis/google-cloud-python/pull/8180/commits/97bf3d2d88f8a4d4afb69e212dd1a47ddee9e31a).

[the remark this answers is not in the document]
Comment (sangramql) [reply bigtable/google/cloud/bigtable/client.py]: @sduskis,
We need to match the different possibilities of transport parameter, which is expected in class [bigtable_table_admin_client.py](https://github.com/googleapis/google-cloud-python/blob/5932539e220a49c462e661115dec775dd10f4f0b/bigtable/google/cloud/bigtable_admin_v2/gapic/bigtable_table_admin_client.py#L192) as you have already pointed out.
Same in classes bigtable_instance_admin_client.py and bigtable_client.py.
Hence I an creating the transport as wrapper to channel and matching the different probabilities like, transport as callable method, as wrapper to channel, as instance to grpctransport instance.

My understanding is that channel is wrapped in transport so i am using for both endpoints.
Do you mean that default channel inside the transport has to used and so channel should not be passed to transport?
Hope i am understanding the issue properly.

[the remark this answers is not in the document]
Comment (sduskis) [reply bigtable/google/cloud/bigtable/client.py]: @sumit-ql, I think something is lost in translation.  I'm going to suggest that we close this PR.  I would also suggest that we work with @tseaver to define the requirements better.

**Pull request closed without merging**